### DISCUSSION TITLE
Server setup

### DISCUSSION THREAD

**Discussion by AkosFarkas07** [Michael-128/qBitControl#56] [Q&A] [answered]
Hi, 

Sorry for the totally beginner question, but I am struggling to going through to the "servers" screen. 
How can I set up a server? 
I just would like to
![IMG_0022](https://github.com/user-attachments/assets/c7e791a5-3eec-4f2c-9aba-885fdcad13c2)
 open the .torrent file from "Files" and download the targetfile to "Files" as well.
Why do I have to setup a server?

What do I miss here?

**Comment by Michael-128** [comment, marked as answer]
qBitControl is an app that allows remote management of a qBittorrent client. For example if you install qBittorrent on your Mac/PC/Server you can manage downloads from your iPhone, but every file will be downloaded and saved on the computer where qBittorrent is installed. qBitControl doesn’t download torrents by itself

**Reply by AkosFarkas07** [reply]
ok, I see.
In this case how can I set up my mac as the server?

Thank you.

**Reply by Michael-128** [reply]
If you install qBittorrent https://www.qbittorrent.org/download you can enable WebUI in the settings and connect via qBitControl
